schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INTEGER GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(100),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg REAL,
```

Comment:
- [ ] [Optional] It would be better if you used `boolean` data type instead of `real` data type.

Reply:
@Omar-Muhamad `boolean` instead of `real`!!!